pkg/codegen/docs/gen.go

```diff
@@ -137,15 +137,16 @@ type header struct {
 
 // property represents an input or an output property.
 type property struct {
+	ID string
```

Comment:
Can you please add a comment for this property? Feel like it is an important detail as to how this is used in the template.

Reply:
good idea.